charts/cortex-agent/templates/_helpers.tpl

```diff
@@ -83,3 +83,14 @@ Create the name of the service account to use
 {{- default "default" .Values.serviceAccount.name }}
 {{- end }}
 {{- end }}
+
+{{/*
+Create the path of the /var/log volume mount on the host fs
+*/}}
+{{- define "cortex-xdr.xdr_var_log_host_path" -}}
```

Comment:
`XdrVarLogHostPath`, let's keep it in the same format as the other fields in the file

Reply:
Agree